tasks/WhereIsThis.tex

```diff
@@ -32,28 +31,18 @@ \subsection*{Additional rules and remarks}
 		\item \textbf{Follower's assistance:} Helping the robot while guiding a guest (e.g.~waving, shouting) causes a score reduction of 50pts.
 	\end{itemize}
 
-	\item \textbf{Information Point:} The robot meets guests and gives directions at the \emph{Information Point}.
-	After guiding a person to its destination, it must return to the information point.
+	\item \textbf{Information Point:} The robot meets guests and gives directions at the \emph{Information Point} and then guides them to the location. The information point is not known beforehand. Right before a run at the entrance door, the team is told a position where they should start the robot.
```

Comment:
I think the info point must be announced 2 hours before the test so the robot enters through the door and reaches it (standard procedure). Two hours is insufficient to hardcode the test.

Reply:
The point is that the start position is randomized for each team. I think it is not too much to ask to be flexible on the spot. Every team in Magdeburg manged that part,.

Reply:
When we tell them the `Information Point` how do we communicate this to them?

1. Point and say "start the robot over there" ?
2. How do they move the robot to the robot their to start ? RViz navigation goal, robot teleop ?

Reply:
I'd say 2 hours before the test. You can hard-code the Info Point position, but you can't really code all directions from there. Besides, is some sort of standard procedure. In addition, I think it's way more fair to have all robots getting questions from the same location.